src/promptflow/promptflow/_utils/utils.py

```diff
@@ -366,3 +366,9 @@ def copy_file_except(src_dir, dst_dir, exclude_file):
             src_file_path = os.path.join(root, file)
             dst_file_path = os.path.join(current_dst_dir, file)
             shutil.copy2(src_file_path, dst_file_path)
+
+
+def prepare_folder(path: Union[str, Path]) -> Path:
```

Comment:
give it some docstring and more precise name as it is a util function now.

Reply:
Updated